apps/api/v2/src/ee/overlay-calendars/inputs/get-overlay-calendars-busy-times.input.ts

```diff
@@ -0,0 +1,28 @@
+import { Type } from "class-transformer";
+import { IsNumber, IsString, IsOptional, IsArray, ValidateNested } from "class-validator";
+
+export class Calendar {
+  @IsNumber()
+  credentialId!: number;
+
+  @IsString()
+  externalId!: string;
+}
+
+export class GetOverlayCalendarsBusyTimesInput {
+  @IsString()
+  loggedInUsersTz!: string;
+
+  @IsString()
+  @IsOptional()
+  dateFrom!: string | null;
```

Comment:
For dateFrom and dateTo - what is the expected date shape? Is it `day-month-year` or `day/month/year` ? Maybe we can create a custom validator like `create-user.input.ts` has `IsTimeFormat`, but in case of dateTo and dateFrom it would check date passed and if its not good respond with the appropriate format.

Reply:
this is the format 2024-02-07